nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -637,7 +644,7 @@ in {
     services.nginx = lib.mkIf cfg.configureNginx {
       enable = true;
       recommendedProxySettings = true; # required for redirections to work
-      virtualHosts."${cfg.localDomain}" = {
```

Comment:
Some form of migration is required for existing users. What and how?

Reply:
Default `webDomain` to `localDomain`?